api/src/main/resources/workbench-api.yaml

```diff
@@ -0,0 +1,6369 @@
+---
+swagger: '2.0'
+info:
+  version: 0.1.0
+  title: AllOfUs Workbench API
+  description: The API for the AllOfUs workbench.
+  termsOfService: http://www.pmi-ops.org/terms_of_service.html
+  contact:
+    name: developer_help@pmi-ops.org
+  license:
+    name: BSD
+host: api.pmi-ops.org
+securityDefinitions:
```

Comment:
A comment explaining the significance of this security definition seems important to carry over, namely that it triggers the auth interceptor.